src/Microsoft.DotNet.CodeFormatter.Analyzers/OptimizeNamespaceImportsAnalyzer.cs

```diff
@@ -9,6 +9,7 @@
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Options;
```

Comment:
yes. these types should be pushed into Roslyn (after getting reviewed/refined/accepted)